Build not working with SSR adapter

Hi!
Maybe this is a known issue, I haven't found anything among the solved issues.

The build does not work with an SSR adapter, I'm using Cloudlfare  and I get this error:

```
[ERROR] [vite] x Build failed in 2.44s
[commonjs--resolver] [plugin vite:resolve] Cannot bundle Node.js built-in "perf_hooks" imported from "node_modules/linkedom/commonjs/perf_hooks.cjs". Consider disabling ssr.noExternal or remove the built-in dependency.
```

Any suggestions?

I appreciate your help

--- a/.changeset/beige-llamas-visit.md
+++ b/.changeset/beige-llamas-visit.md
@@ -1,0 +1,5 @@
+---
+"@astro-community/astro-embed-utils": patch
+---
+
+Fixes support for serverless usage, for example with the Cloudflare adapter.

--- a/packages/astro-embed-utils/index.ts
+++ b/packages/astro-embed-utils/index.ts
@@ -1,4 +1,4 @@
-import { parseHTML } from 'linkedom';
+import { parseHTML } from 'linkedom/worker';
 
 class LRU<K, V> extends Map<K, V> {
 	constructor(private readonly maxSize: number) {

